source/server-discovery-and-monitoring/tests/unified/serverMonitoringMode.yml

```diff
@@ -171,3 +171,44 @@ tests:
       # Wait for the second serverHeartbeatStartedEvent to ensure we do not stream.
       - *waitForSecondHeartbeatStarted
     expectEvents: *pollingStartedEvents
+
+  - description: "poll waits after successful heartbeat"
+    operations:
+      - name: createEntities
+        object: testRunner
+        arguments:
+          entities:
+            - client:
+                id: client
+                uriOptions:
+                  serverMonitoringMode: "poll"
+                  heartbeatFrequencyMS: 1000000
+                useMultipleMongoses: false
+                observeEvents:
+                  - serverHeartbeatStartedEvent
+                  - serverHeartbeatSucceededEvent
+                  - serverHeartbeatFailedEvent
```

Comment:
Can remove this since we don't care about the heartbeat failed events

Reply:
Done.